pennylane/pulse/parametrized_evolution.py

```diff
@@ -393,6 +396,7 @@ def __init__(
         self.hyperparameters["return_intermediate"] = return_intermediate
         self.hyperparameters["complementary"] = complementary
         self._check_time_batching()
+        self.dense = dense or len(self.wires) < 3
```

Comment:
I don't personally like to rely on the implicit False-ness of `None` so I'd suggest this more explicit usage, but I don't know what's the standard style in PL..

```suggestion
        if dense is None:
            dense = len(self.wires) < 3
        self.dense = dense
```

Reply:
This is quite standard in PL

Reply:
Wait, but this does not work if we actually want `dense=False` to be a valid value of the variable, right? :sweat_smile: 
```suggestion
        self.dense = len(self.wires) < 3 if dense is None else dense
```

Reply:
please ignore my ignorance, you are both right 🙈